### ISSUE TITLE
Feedback: Remove placeholder text in search box

### ISSUE THREAD

**Issue by afeijoo** [18F/govt-wide-patternlibrary#117]
>Placeholder text within a form field makes it difficult for people to remember what information belongs in a field, and to check for and fix errors. It also poses additional burdens for users with visual and cognitive impairments.

Placeholders in Form Fields Are Harmful, Nielsen Norman Group
http://www.nngroup.com/articles/form-design-placeholders/

See also http://www.digitalgov.gov/2014/11/24/placeholder-text-think-outside-the-box/

**Comment by mollieru** [comment]
+ @emilyville + @dannychapman who had suggested this to start. Did you guys have research or data behind this recommendation?

**Comment by dannychapman** [comment]
Moving form field labels inside a field (not to be confused with placeholder text) does show worse completion times (Luke W. has data to support this, and I'm sure he's not alone), but placeholder text can be used effectively as another tool to help with completion. The first link listed here supports this too. I think simply supporting this practice of using it in addition to, and never in place of, might be a good place to start.

**Comment by colinpmacarthur** [comment]
I think we should probably try to avoid placeholder text whenever we can. Check out http://www.nngroup.com/articles/form-design-placeholders/, also linked above.

TLDR: Really not good to rely on placeholder text as the only label for a field, but also not great to have placeholder text with additional information. The big issue is that folks have to remember what those placeholders said after they enter something into the field.

**Comment by maya** [comment]
Do we want to consider providing an alternate label style for hints?
![label-hint-outside](https://cloud.githubusercontent.com/assets/5249443/8538721/b6775d44-2425-11e5-921b-decfaf90abf2.png)

**Comment by carodew** [comment]
@maya that looks good to me. Do you need anything from us or @angel to implement that?

**Comment by maya** [comment]
@carodew text size and shade of gray should be all I need.

**Comment by angel** [comment]
@maya I would say $color-gray-600 (#757575, the lightest gray we're using that passes contrast with white background) and 14px (.875em). What do you think @carodew?

**Comment by carodew** [comment]
Re color and size - sounds good, let's do it and we'll see what feedback we get from user testing. :)

**Comment by colinpmacarthur** [comment]
Looks like we have an alternative approach for text field hints (other than placeholder text). @maya: Has this been implemented? Or shall I put it in the icebox?

**Comment by maya** [comment]
@colinpmacarthur we haven't started building out forms, so not yet.

**Comment by colinpmacarthur** [comment]
I will add this to the project workflow. Thanks for the good discussion, everyone!
